src/interface/src/app/treatments/treatments-tab/treatments-tab.component.ts

```diff
@@ -67,8 +67,9 @@ export class ProjectAreaTreatmentsTabComponent {
   }
 
   setSearchString(searchString: string) {
-    this.searchString = searchString;
-    if (this.searchString === '') {
+    this.searchString = searchString.toLowerCase();
+    if (this.searchString === '' || this.searchString === null) {
```

Comment:
I see there are several places where we are checking for this two things, either empty string or null. Wondering if we can streamline and only have one of those two use cases. Might simplify the code downstream